SPECS-EXTENDED/acpid/acpid.spec

```diff
@@ -101,6 +101,10 @@ fi
 	/bin/systemctl try-restart acpid.service >/dev/null 2>&1 || :
 
 %changelog
+
```

Comment:
Can you please verify the licenses by updating the corresponding markdown file using the script license_map.py? Once you have completed this task, kindly add an entry to the changelog with "License verified" or "Verified license".

Reply:
updated